exercises/pascals-triangle/example.h

```diff
@@ -0,0 +1,13 @@
+#if !defined(PASCALS_TRIANGLE_H)
+#define PASCALS_TRIANGLE_H
+
+#include <vector>
+
+namespace pascals_triangle
+{
+
+std::vector<std::vector<int>> gen(int n);
```

Comment:
I don't like the name of this function, we should avoid using abbreviations if we can.

Maybe `generate` or even `generate_rows`?

Also, please use a `size_t` as your input instead of an int.

Reply:
Updated